packages/next-server/types/index.d.ts

```diff
@@ -3,8 +3,8 @@ import {
   NextComponentType,
   NextApiResponse,
   NextApiRequest,
-} from 'next-server/lib/utils'
-import { RouterUrl } from 'next-server/lib/router/router'
+} from 'next-server/dist/lib/utils'
+import { RouterUrl } from 'next-server/dist/lib/router/router'
```

Comment:
Should these be relative paths given that the import is from the same package? 🤔

Reply:
I'm removing these types, they should only be in the `next` package